coderd/workspaceagents.go

```diff
@@ -755,31 +755,69 @@ func convertWorkspaceAgent(derpMap *tailcfg.DERPMap, coordinator tailnet.Coordin
 func (api *API) workspaceAgentReportStats(rw http.ResponseWriter, r *http.Request) {
```

Comment:
This should have tests that check the stat gets updated in the db and that the last used at gets updated properly